tests/Models/UserTest.php

```diff
@@ -239,14 +239,16 @@ public function testUsernameChangeCostWindow(int $years, int $cost)
         $this->travelTo(CarbonImmutable::now()->subYears(3));
 
         $user = User::factory()->create();
-        while (CarbonImmutable::now()->isBefore($now)) {
+        // every 6 months for 3 years = 6
+        // using isBefore to setup adds too many when run at month boundaries.
+        for ($i = 0; $i < 6; $i++) {
```

Comment:
although it's not for another three years, there's a vaguely similar problem when run on feb 29

Reply:
it ironically works fine because `usernameChangeCost` doesn't check if the date is in the future

Reply:
still failing with `$now = CarbonImmutable::now()->subYears(1)->startOfMonth()->subDays(1);`...